utils/models_to_deprecate.py

```diff
@@ -0,0 +1,163 @@
+"""
+Script to find a candidate list of models to deprecate based on the number of downloads and the date of the last commit.
+"""
+import argparse
+import glob
+import json
+import os
+from collections import defaultdict
+from datetime import datetime, timezone
+from pathlib import Path
+
+from git import Repo
+from huggingface_hub import HfApi
+
+
+api = HfApi()
+
+PATH_TO_REPO = Path(__file__).parent.parent.resolve()
+repo = Repo(PATH_TO_REPO)
+
+
+def _extract_commit_hash(commits):
+    for commit in commits:
+        if commit.startswith("commit "):
+            return commit.split(" ")[1]
+    return ""
+
+
+def get_list_of_repo_model_paths(models_dir="src/transformers/models"):
```

Comment:
Probably don't specify the default value here, but specify the value where we call this method.

(Or just have a variable at the top of this file and use it as default value here)

(I admit it's more a taste not a real issue.)

Reply:
Default removed and a posix path passed in from the main function in [98183b9](https://github.com/huggingface/transformers/pull/29686/commits/98183b9901e0828be6763dd280d45fbb473aba39)